LICENSE

```diff
@@ -307,11 +286,6 @@ For details see http://jquery.org/license
 
```

Comment:
Do you need to change this to point to the license file you copied to licenses directory?